www/templates/metrics/metrics-control.html

```diff
@@ -1,6 +1,6 @@
 <ion-modal-view style="background-color: #eeeeee">
 	<ion-header-bar>
-		<button side="left" class="button" ng-click="closeModal()">Cancle</button>
+		<button side="left" class="button" ng-click="closeModal()">Cancel</button>
```

Comment:
Thank you so much for finally fixing this eyesore.

Reply:
No problem, just happened to spot it.